### ISSUE TITLE
--output-path used with multiple output types on GitLab CI doesn't function as documented.

### ISSUE THREAD

**Issue by Benthescott** [GoogleChrome/lighthouse#10101]
According to the readme, --output-path when used with multiple output formats works like this:
> If using multiple outputs, --output-path is appended with the standard extension for each output type. "reports/my-run" -> "reports/my-run.report.html", "reports/my-run.report.json", etc.

Using the CLI on my local works exactly as documented. However, when run on GitLab CI, it only creates the file format specified in --output-path even if multiple types are specified.

**Comment by patrickhulce** [comment]
Could you be more specific about when running in Gitlab CI? i.e. exact commands used in both environments, Lighthouse versions of each environment, all the information in the [bug report template](https://github.com/GoogleChrome/lighthouse/issues/new?template=Bug_report.md), etc

**Comment by connorjclark** [comment]
@Benthescott could you provide more info?

**Comment by Benthescott** [comment]
#### Provide the steps to reproduce
`npm i -g lighthouse`
```bash
lighthouse https://www.google.com --quiet --chrome-flags="--headless --no-sandbox" \
      --output json --output html --output-path ./lighthouse \
      --only-categories=performance,accessibility,best-practices,seo
```
#### What is the current behavior?
creates only an html report called 'lighthouse' with no file extension

#### What is the expected behavior?
Creates both a json and html reports named 'lighthouse.report.json' and 'lighthouse.report.html'

#### Environment Information
* Affected Channels: NodeJS CLI tool
* Lighthouse version: Latest
* Chrome version: 77
* Node.js version: 10.16.0
* Operating System: cypress/browsers:node10.16.0-chrome77 docker image, which is based on the official node images that use Debian.

Let me know if you need any more info, sorry for the late response.

**Comment by connorjclark** [comment]
Strange, that should work. Same result if you use `--output json,html --output-path lighthouse`?

**Comment by Benthescott** [comment]
Same behavior on GitLab, however locally I got
`Invalid values:
  Argument: output, Given: "json,html", Choices: "json", "html", "csv"`

**Comment by connorjclark** [comment]
Ah, the csv option doesn't work for `output` (bug). I expected it would.

I'm thinking the arg parsing isn't working for some reason... Can you provide a repo with a Dockerimage? otherwise, can you print out `argv` in `cli-flags`?

**Comment by brendankenny** [comment]
The arg parsing with commas was fixed in #10188, but since there hasn't been any movement on this issue since January, I'm going to close for now. If this is still an issue you're encountering, please comment and we can reopen!
